Resources/Locale/en-US/prototypes/roles/antags.ftl

```diff
@@ -32,4 +32,7 @@ roles-antag-evil-twin-description = Replace your twin.
 roles-antag-minor-antag-name = Minor Antag
 roles-antag-minor-antag-objective = Make your way through a normal shift, considering your ulterior motives.
 
+roles-antag-wizard-antag-name = Wizard
+roles-antag-wizard-objective = The Wizard Federation has selected you to eliminate the crew.
+
```

Comment:
These should go in the SimpleStation FTL folder.
Should be Resources/Locale/en-US/SimpleStation14/ if I'm not mistaken.